### DISCUSSION TITLE
Can't install in Centos 7

### DISCUSSION THREAD

**Discussion by okgame** [uNetworking/uWebSockets.js#387] [Q&A] [answered]
Hello,
I got issue when install uWebsocket in Centos 7.
I tried install lasted version of uWebsocket and old version, all got same issue.
Issue: https://i.imgur.com/SlQ3tc3.png

Centos 7 use default glibc 2.17.
I tried upgrade glibc to 2.18 and it fail, my OS got issue too.
Have any idea run uWebsocket without upgrade glibc? It too risk.

**Comment by ghost** [comment, marked as answer]
Centos 8 works. Other than this, no, you can't cover every single (outdated) dist by default. This is not a real problem since we have Docker where any dist can be selected easily
